docs/hyperopt.md

```diff
@@ -246,12 +251,11 @@ use data from directory `user_data/data`.
 
 ### Running Hyperopt with Smaller Testset
 
-Use the `--timerange` argument to change how much of the testset
-you want to use. The last N ticks/timeframes will be used.
-Example:
+Use the `--timerange` argument to change how much of the testset you want to use.
+To use one month of data, use the following parameter:
```

Comment:
'For instance, to use...' (or restore 'Example:' here)

(too many 'use' words in these 2 lines...)

Reply:
The rest is fine, nice work. 👍